### PR TITLE
fix: trim routes for android - no more then 6 tabs

### PR DESCRIPTION
**Pull request by mrpmohiburrahman**
This PR fixes issue https://github.com/okwasniewski/react-native-bottom-tabs/issues/13

Right now it will show a warning if there are more than 6 tabs on android and will only show 6 tabs and trims the other tabs.

https://github.com/user-attachments/assets/4ebfbffa-2c26-4332-b3b8-35d6337206db

### CODE REVIEW

File: src/TabView.tsx [hunk 1]
```diff
@@ -87,9 +89,15 @@ const TabView = <Route extends BaseRoute>({
...
-  }
+  const trimmedRoutes = useMemo(() => {
+    if (Platform.OS === 'android' && navigationState.routes.length > MAX_TABS) {
+      console.warn(`TabView only supports up to ${MAX_TABS} tabs on Android`);
+      return navigationState.routes.slice(0, MAX_TABS);
+    }
+    return navigationState.routes;
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (okwasniewski) [inline src/TabView.tsx:100]: You shouldn't ignore this rule. It should be recalculated when navigation state changes (for example someone dynamically adds a route)

File: src/TabView.tsx [hunk 2]
```diff
@@ -70,6 +70,8 @@ interface Props<Route extends BaseRoute> {
   }) => ImageSource | undefined;
 }
 
+const MAX_TABS = 6;
```
Comment (okwasniewski) [inline src/TabView.tsx:73]: Let's name it `ANDROID_MAX_TABS`

File: src/TabView.tsx [hunk 3]
```diff
@@ -176,7 +182,7 @@ const TabView = <Route extends BaseRoute>({
           <View
             key={route.key}
             style={[
-              { width: '100%', height: '100%' },
+              styles.sceneContainer,
```
Comment (okwasniewski) [inline src/TabView.tsx:185]: You can re-use the fullWidth style, no need to create the same one

**Comment by mrpmohiburrahman** [comment]
Hi, @okwasniewski , 
Thanks for the reviews.
I fixed those.

**Review by okwasniewski — APPROVED** [review]
Thanks! Great work 🙌

**Comment by mrpmohiburrahman** [comment]
> Thanks! Great work 🙌

I'm excited to contribute further.

Please let me know if there are any other issues I can assist with.

**Comment by okwasniewski** [comment]
> > Thanks! Great work 🙌
> 
> I'm excited to contribute further.
> 
> Please let me know if there are any other issues I can assist with.

Awesome! This could be a good issue to work on next: https://github.com/okwasniewski/react-native-bottom-tabs/issues/15

**Pull request merged**